promoter-site-health.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Plugin Name: Promoter Site Health
+ * GitHub Plugin URI: https://github.com/mt-support/promoter-site-health
+ * Author:            Modern Tribe, Inc.
+ * Author URI:        http://m.tri.be/1971
+ * Description: Add site health checks to make sure Promoter is working properly.
+ * Requires PHP: 7.0
+ * Text Domain: tribe_extensions_promoter
```

Comment:
`tribe_extensions_promoter` should match plugin slug (folder name) here and throughout, e.g. `tribe-ext-promoter-site-health`

This should also be this repo's name and the main plugin file's name.

Additionally, all the `__()` should be `esc_html__()` to protect from HTML via translations.

Reply:
I think we can remove the `tribe-ext` prefix as because the nature of this organization this implies this repo is an extension. so I can update it to `promoter-site-health`

Reply:
This will get released as an Extension, right?
By "nature of this organization", do you mean "github.com/mt-support"? If not, which "organization" do you mean?

`tribe-ext` prefix is so it loads in the proper order and can use the Extension's functionalities, like requiring certain core plugins and of a minimum version number -- https://github.com/moderntribe/tribe-common/blob/4.12.8/src/Tribe/Extension_Loader.php#L54

Reply:
Yes the GH organization "github.com/mt-suppo" and this is going to be relased as an extension it behaves as an isolated plugin as it does not uses any of the logic of extension to be executed or loaded as it only uses a native WP action to be executed instead.

https://github.com/mt-support/promoter-site-health/blob/v1/promoter-site-health.php#L15-L16

So it does not require to be loaded via the `common` module.

Reply:
what about _some_ sort of namespace/prefix, such as tribe-promoter-site-health (but in that case why not tribe-ext-... ---- no personal preference from me